pkg/operator/ceph/cluster/utils.go

```diff
@@ -32,26 +32,61 @@ import (
 // Its content allows modifying Ceph configuration flags
 func populateConfigOverrideConfigMap(clusterdContext *clusterd.Context, namespace string, ownerInfo *k8sutil.OwnerInfo) error {
 	ctx := context.TODO()
-	placeholderConfig := map[string]string{
-		k8sutil.ConfigOverrideVal: "",
+	// Add the helm labels in case the user wants to install the cluster helm chart to start managing it
+	labels := map[string]string{
```

Comment:
And A Doubt the other daemons are still labeled by `labels["app.kubernetes.io/managed-by"] = "rook-ceph-operator"` so if we need to update them if installed by helm?

Reply:
We're just labeling this one configmap, so this won't impact the other places that label is applied.